src/main/rx/rx.c

```diff
@@ -862,3 +862,12 @@ bool isRssiConfigured(void)
 {
     return rssiSource != RSSI_SOURCE_NONE;
 }
+
+bool rxGetFrameDelta(timeDelta_t *deltaUs)
+{
+    if (rxRuntimeState.rcFrameDeltaFn) {
+        *deltaUs = rxRuntimeState.rcFrameDeltaFn();
```

Comment:
Would it make sense to change this to `timeDelta_trxGetFrameDelta(void)` and return 0 if no function is defined for the protocol?

Reply:
Eh, I think it's six of one, half a dozen of the other. Was trying to be literal about the "not available" part rather than assuming that a 0 result implied that. But since 0 isn't really a possible valid value I don't have a objection to doing it this way.

Reply:
After thinking about this for a bit I'm going to change my position and say I prefer the boolean explicit availability test. The reason is that it aids debugging and testing as the functionality is implemented for additional protocols. When using the boolean method we can clearly see if the implementation is working properly by simply viewing the output of `rc_smoothing_info`. However using the zero-result method will mask implementation errors by potentially falling back to the high-level calculation. The problem is we would have no way of know whether it's falling back to the previous calculation method. Having an explicit view into the delay calculation will make it easier to confirm results in cases where we have to rely on user testing because we don't have access to the relevant hardware to verify during development.

Reply:
Not sure I follow the reasoning, as we could have the function return 0 as a special case, and detect and report this at the higher level just as well as we do with having an explicit state - I think it is unlikely that we will ever find an RC system that can receive multiple RC frames in less than a microsecond.
But as you say, this is six of one, half a dozen of the other.